<?xml version="1.0"?>
<launch>
    <rosparam command="load" file="$(find mvbb)/config/calibration.yaml" ns="/mvbb"/>
    <node pkg="mvbb" type="mvbb_node" name="mvbb" output="screen">
        <param name="cluster_tolerance" value="0.02"/>
        <param name="cluster_min_size" value="100"/>
        <param name="cluster_max_size" value="70000"/>
        <param name="input_topic" value="/pacman_vision/processed_scene"/>
        <param name="reference_frame" value="world"/>
        <param name="invert_z_projection" value="true"/>
        <param name="broadcast_results" value="true"/>
    </node>
</launch>